ceph/UI/operations.py

```diff
@@ -0,0 +1,24 @@
+from dashboard import Dashboard
+
+
+class Operations:
+    def __init__(self, driver):
+        """Initialize Operations with a WebDriver instance."""
+        self.driver = driver
+        self.dashboard = Dashboard(driver)
+
+    def login(self, username: str, password: str):
+        """Performs a login operation using the provided credentials."""
+        try:
+            username_field = self.driver.find_element_by_id("username")
+            password_field = self.driver.find_element_by_id("password")
+            login_button = self.driver.find_element_by_id("login")
```

Comment:
Add logic to make it generic. Not always its the id, it can accept all other identifiers.